components/engine_rocks/src/tests.rs

```diff
@@ -0,0 +1,144 @@
+// Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
```

Comment:
Is it necessary to move the tests to a separate file? seems it's not a TiKV convention.

Reply:
I've moved them to engine.rs.

Reply:
I think it is ok.